src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts

```diff
@@ -143,7 +143,7 @@ function getMatchingRootRouteForRHPRoute(
 
     // Check for FullScreenNavigator
     for (const [fullScreenName, RHPNames] of Object.entries(FULL_SCREEN_TO_RHP_MAPPING)) {
-        if (RHPNames && RHPNames.includes(route.name)) {
```

Comment:
Do we no longer need to safely check this? Or is this more of a lint concern? Maybe we could do something like RHPNames?.includes(route.name) if it's the lint concern?

Reply:
Pretty sure this change is safe as-is...

`FULL_SCREEN_TO_RHP_MAPPING` is a `Partial<Record<CentralPaneName, string[]>>`, which means that all the values of the object are `string[]`, which can't be null or undefined and should always have a `includes` property.

Reply:
Ah cool, also didn't see that `FULL_SCREEN_TO_RHP_MAPPING` is constant.